cache/modules/wc_org_cloudfront/distribution.tf

```diff
@@ -0,0 +1,188 @@
+  enabled          = true
+  retain_on_delete = false
+  is_ipv6_enabled  = true
+  aliases          = var.aliases
+
+  // The primary load balancer for wc.org apps
+  origin {
+    domain_name = var.load_balancer_dns
+    origin_id   = local.alb_origin_id
+
+    custom_origin_config {
+      origin_protocol_policy = "https-only"
+      http_port              = "80"
+      https_port             = "443"
+      origin_ssl_protocols   = ["TLSv1.2"]
+    }
+  }
+
+  // The S3 bucket containing some static assets
+  origin {
+    domain_name = var.assets_origin.bucket_endpoint
+    origin_id   = local.assets_origin_id
+  }
+
+  # Don't cache 404s
+  custom_error_response {
+    error_code            = 404
+    error_caching_min_ttl = 0
+  }
+
+  viewer_certificate {
+    acm_certificate_arn      = var.certificate_arn
+    ssl_support_method       = "sni-only"
+    minimum_protocol_version = "TLSv1.2_2018"
+  }
+
+  restrictions {
+    geo_restriction {
+      restriction_type = "none"
+    }
+  }
+
+  logging_config {
+    include_cookies = false
+    bucket          = "wellcomecollection-experience-cloudfront-logs.s3.amazonaws.com"
+    prefix          = "wellcomecollection.org/${var.namespace}/"
+  }
+
+
+  # The default cache behaviour is to forward requests to the content app
+  default_cache_behavior {
+    target_origin_id = local.alb_origin_id
+
+    allowed_methods        = local.all_methods
+    cached_methods         = local.stateless_methods
+    viewer_protocol_policy = "redirect-to-https"
+
+    cache_policy_id          = var.cache_policies["weco-apps"]
+    origin_request_policy_id = var.request_policies["host-query-and-toggles"]
+
+    dynamic "lambda_function_association" {
+      for_each = local.lambda_associations
+      content {
+        event_type = lambda_function_association.value.event_type
+        lambda_arn = lambda_function_association.value.lambda_arn
+      }
+    }
+  }
+
+  # Works
+  ordered_cache_behavior {
+    path_pattern     = "/works*"
+    target_origin_id = local.alb_origin_id
+
+    allowed_methods        = local.stateless_methods
+    cached_methods         = local.stateless_methods
+    viewer_protocol_policy = "redirect-to-https"
+
+    cache_policy_id          = var.cache_policies["weco-apps"]
+    origin_request_policy_id = var.request_policies["host-query-and-toggles"]
+
+    dynamic "lambda_function_association" {
+      for_each = local.lambda_associations
+      content {
+        event_type = lambda_function_association.value.event_type
+        lambda_arn = lambda_function_association.value.lambda_arn
+      }
+    }
+  }
+
+  # Account
+  ordered_cache_behavior {
+    path_pattern     = "/account*"
+    target_origin_id = local.alb_origin_id
+
+    allowed_methods        = local.all_methods
+    cached_methods         = local.stateless_methods
+    viewer_protocol_policy = "redirect-to-https"
+
+    cache_policy_id          = var.cache_policies["Managed-CachingDisabled"]
```

Comment:
I assume this is to avoid sending responses for one user to another user? Let's call that out in a comment here. Getting this one wrong would be more serious than, say, mucking up the works policy.

Reply:
Indeed, good point